deploy/ansible/worker/include/run_geth.yml

```diff
@@ -0,0 +1,47 @@
+- name: "Sync/Run Geth"
```

Comment:
On mainnet, this play will need to stand alone for a multi-day syncing time.

Reply:
Maybe we add a flag or something in inventory.yml to allow for light clients?

Reply:
@mswilkison this is available already through the geth_options envvar in the inventory.  It defaults to`—goerli` for cassandra